README.md

```diff
@@ -5,8 +5,8 @@ Alacritty
 
 Alacritty is the fastest terminal emulator in existence. Using the GPU for
 rendering enables optimizations that simply aren't possible in other emulators.
-Alacritty currently supports FreeBSD, Linux, and macOS. Windows support is planned
-before the 1.0 release.
+Alacritty currently supports FreeBSD, Linux, Mac OS, and OpenBSD. Windows
```

Comment:
The `macOS` string was actually correct here. Apple changed the formatting a while back to match iOS and tvOS.

Reply:
Ha!  Wow.  I hadn't realized that the name changed yet again!  Product managers make about as much needless churn as daylight savings rules.  I've reverted this bit back now.